cluster-scope/overlays/nerc-ocp-test/machineconfigs/pci-passthrough/100-4xlag-iommu.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: 4xlag
```

Comment:
Might be better to just use the worker role instead of `4xlag` for now. That pool is a stop-gap solution to configure LACP with 4x NICs vs 2x NICs (default). I'm still experimenting with ways to avoid having to create a custom MCP and also haven't made a PR for that piece yet (or the udev rules which are manually in place for now).

Reply:
@jtriley sure, I have updated this PR to use the `worker` role instead.